File: wasm_tests/test_7.js
const VERY_BIG_JSON = await fetch("/wasm_tests/test_7.json").then((x) => {
    return x.text();
});

export function test_7() {
    for (let i = 0; i < 5; i++) {
        JSON.parse(VERY_BIG_JSON);
    }
}

File: wasm_tests/test_7.rs
const VERY_BIG_JSON: &str = include_str!("./test_7.json");

pub fn test_7() {
    for _ in 0..5 {
        std::hint::black_box(serde_json::from_str::<serde_json::Value>(VERY_BIG_JSON).unwrap());
    }
}
